content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+
+### [](#create-a-bucket-in-hpe-ezmeral-data-fabric-object-store-and-upload-some-objects)Create a bucket in HPE Ezmeral Data Fabric Object Store and upload some objects
+
+In order to create a Bucket, you need to create an Account, an IAM User, and finally a Bucket in sequence.
+While creating IAM User and Bucket, you also need to prepare Access Policy to control who (IAM User) can perform which (list, get, put, etc.) operations on which Bucket.
+
+You can read the following document - [Entities and Resources](https://docs.datafabric.hpe.com/72/MapROverview/object_store_account_management.html) first to gain a deeper understanding of the entity model of HPE Ezmeral Data Fabric Object Store.
+
+Below, I will demonstrate in turn how to create the above-required entities.
+
+#### Create an account
+
+I choose to use the [mc](https://docs.datafabric.hpe.com/72/ReferenceGuide/mc-commands-overview.html) command line tool to create Account.
+
+Before using the mc command line for the first time, you need to create an Alias for your administrator.
+
+An Alias contains an access endpoint, such as "https://s3-us-west-1.amazonaws.com", which is an Amazon AWS S3 endpoint; another example is "http://10.10.88.198:9000", which is a Minio endpoint.
+An Alias also contains the Access Key and Secret Key used by your administrator or IAM User.
+
+##### 1. You first use the "[mc alias list](https://docs.datafabric.hpe.com/72/ReferenceGuide/mc-alias-list.html)" command to view the default Alias in the following systems.
+
+❗Note: If you are using self-signed TLS certificates or installed the cluster by Installer, you have to copy <ins>/opt/mapr/conf/ca/chain-ca.pem</ins> to <ins>~/.mc/certs/CAs/</ins> on the node running `mc`.
+The meaning of this step is the same as you imported the self-issued CA to the keytool of the JVM earlier, `mc` also needs to import the self-issued CA to communicate with the S3server of the Object Store.
+
+```shell
+sudo -E -u mapr /opt/mapr/bin/mc alias list
+```
+
+Sample output:
+
+```
+gcs
+  URL       : https://storage.googleapis.com
+  AccessKey : YOUR-ACCESS-KEY-HERE
+  SecretKey : YOUR-SECRET-KEY-HERE
+  API       : S3v2
+  Path      : dns
+
+local
+  URL       : http://localhost:9000
+  AccessKey :
+  SecretKey :
+  API       :
+  Path      : auto
+
+play
+  URL       : https://play.min.io
+  AccessKey : XXXXXXXXXXXXXXXXXXXX
+  SecretKey : XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  API       : S3v4
+  Path      : auto
+
+s3
+  URL       : https://s3.amazonaws.com
+  AccessKey : YOUR-ACCESS-KEY-HERE
+  SecretKey : YOUR-SECRET-KEY-HERE
+  API       : S3v4
+  Path      : dns
+```
+
+##### 2. Generate S3 keys to authenticate your administrator

```

Comment:
Double check this in preview mode. I'm not sure you want these bullets to be this small.